jupyter chap03, flow edits chap02-chap03 (#186)
datacarpentry/image-processing: _episodes/03-skimage-images.md

JPEG into a NumPy array, and returns the array; we save the array in a variable
named `image`.

Next, we will do something with the image:

~~~
fig, ax = plt.subplots()
plt.imshow(image)
~~~
{: .language-python}

> Once we have the image in the program, we first call `plt.subplots()` so that we will have a fresh figure with a set of axis independent from our previous calls. Next we call `plt.imshow()` in order to display the image.

Reviewer: ```suggestion
Once we have the image in the program, we first call `plt.subplots()` so that we 
will have a fresh figure with a set of axis independent from our previous calls. 
Next we call `plt.imshow()` in order to display the image.
```

Splitting up the long line

> ## Why not use `skimage.io.imshow()`
>

> >  The *skimage* library has its own function to display and image, so you might

Reviewer: ```suggestion
>  The *skimage* library has its own function to display an image, so you might
```

> be asking why we don't use it here.  It is certainly something you should be aware of 
> and may use as you see fit in your own code, but the details of what it will do to 
> display the image is currenlty in the process of change. Thus, calling `imshow()` off the 

> > *matplotlib.pyplot* library at this time insures participants have the 

Reviewer: ```suggestion
> *matplotlib.pyplot* library at this time ensures participants have the 
```

> > experience we need across platforms for this course, so we will be doing that instead.

Reviewer: ```suggestion
> experience we need across platforms for this lesson, so we will be doing that instead.
```

Mostly, The Carpentries uses "workshop" and "lesson" instead of "course"

>
> ![Su-Do-Ku puzzle](../data/sudoku.png)
>

> > Replicate the ignore low intensity script above as a starting. Then, edit the script 

Reviewer: ```suggestion
> As a starting point, replicate the code we used to ignore low intensity pixels above. Then, edit this copy 
```

Author: This one is not longer relevant after picking up changes from gh-pages in previous commit

~~~
"""
* Python script to load a color image as grayscale.
"""
import skimage.io
import skimage.color

# read input image, based on filename parameter
image = skimage.io.imread('data/chair.jpg', as_gray=True)

# display grayscale image
fig, ax = plt.subplots()
plt.imshow(image)
~~~
{: .language-python}

## Access via slicing

> As noted in the previous lesson skimage images are stored as NumPy arrays, so we can use array slicing to select rectangular areas of an image. Then, we can save the selection as a new image, change the pixels in the image, and so on. It is important to

Reviewer: ```suggestion
As noted in the previous lesson skimage images are stored as NumPy arrays, 
so we can use array slicing to select rectangular areas of an image. 
Then, we can save the selection as a new image, change the pixels in the image, and so on. 
It is important to
```

Splitting up the long line

![Whiteboard image](../data/board.jpg)

Using the same display technique we have used throught this course, we can 

> determine the coordinates of the corners of the area we wish to extract by hovering the mouse near the points of interest and noting the coordinates. 

Reviewer: ```suggestion
determine the coordinates of the corners of the area we wish to extract by 
hovering the mouse near the points of interest and noting the coordinates. 
```

Splitting up the long line
